app/src/androidTest/java/com/mutualmobile/barricade/sample/BarricadeActivityTest.java

```diff
@@ -74,7 +78,7 @@
     Espresso.pressBack();
```

Comment:
Also, the styling of buttons for reset dialog is different from that for delay dialog. Personally, I like the styling of the reset dialog.